common/shlibs

```diff
@@ -1794,7 +1794,7 @@ libmatekbdui.so.6 libmatekbd-1.28.0_1
 libmatemixer.so.0 libmatemixer-1.10.0_1
 libmateweather.so.1 libmateweather-1.8.0_1
 libmate-panel-applet-4.so.1 libmate-panel-1.8.0_1
-liblz4.so.1 liblz4-1.8.2_1
+liblz4.so.1 liblz4-1.10.0_1
```

Comment:
this version is the *minimum* version that provides that shlib, unless there is an ABI break.